concepts/05_credentials/01_overview.md

```diff
@@ -17,23 +17,23 @@ import ThemedImage from '@theme/ThemedImage';
 />
 </center>
 
-To get a credential, a Claimer needs to take the following steps:
+To get a credential, a Holder needs to take the following steps:
 
-1. Find a **CType** to base a claim on. Potential Attesters and Verifiers might advertise this information themselves.
+1. Find a **CType** to base a claim on. Potential Issuers and Verifiers might advertise this information themselves.
 2. Make a **claim** containing a set of properties about themselves.
-3. Fulfil any requirement from your Attester. For example, accepting their **Terms** and paying a **Quote**.
-4. **Request an attestation** from the Attester.
-5. Wait for the Attester to **attest** claims.
+3. Fulfil any requirement from your Issuer. For example, accepting their **Terms** and paying a **Quote**.
```

Comment:
I think we should get rid of point 2 and change this line as follows:
```suggestion
2. Fulfil any requirement from your Issuer. This may include providing them with **claims** about yourself that you want to see included in a credential, accepting their **Terms**, and paying a **Quote**.
```